install/0000_30_machine-api-operator_00_credentials-request.yaml

```diff
@@ -151,3 +151,19 @@ spec:
     apiVersion: cloudcredential.openshift.io/v1
     kind: VSphereProviderSpec
 ---
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: openshift-machine-api-kubevirt
+  namespace: openshift-cloud-credential-operator
+spec:
+  secretRef:
+    namespace: openshift-machine-api
+    # This is the of the new secret
```

Comment:
Should be "This is the **name** of the new secret"

Reply:
I removed the comment